purchases/src/main/kotlin/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -760,11 +779,11 @@ internal class Backend(
         }
     }
 
-    private fun <K, S, E> MutableMap<K, MutableList<Pair<S, E>>>.addCallback(
+    private fun <K, F> MutableMap<K, MutableList<F>>.addCallback(
```

Comment:
I didn't need separate error/success callbacks for this operation so had to change this. I could have moved that transformation outside, but it would mean we would need to be leaking the result body out of the backend class, so I thought this was a bit better...